### ISSUE TITLE
Help Pleasee

### ISSUE THREAD

**Issue by apkbir** [Mastaaa1987/vavoo-parser#7]
I need your support, my friend, how can I reach you

**Comment by apkbir** [comment]
I'm trying to install vavoo for my own app, but the video addresses are not playing in the player

**Comment by apkbir** [comment]
yes, it's set up, but it's still not working, please can you help me

**Comment by digitvadmin** [comment]
> yes, it's set up, but it's still not working, please can you help me

Kekik Cloudstream reposuna Vavoo yayınları mı koymayı düşünüyorsunuz ?
